### PR TITLE
fix(curriculum): reset lesson text

### LINKED ISSUE #55354: Reset this lesson doesn't reset on projects
If I press Reset this lesson on final projects, the saved code is restored instead of the starting code.

Can we have two different buttons?

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55354

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
This is looking much better, thanks @gagan-bhullar-tech! To fully close the issue, though, it would be better if the text in the modal became

> Are you sure you wish to revert this lesson? Your latest changes will be undone and the code reverted to the most saved version.

**Comment by gagan-bhullar-tech** [comment]
@ojeytonwilliams updated PR.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Thanks, @gagan-bhullar-tech.

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -52,6 +52,8 @@
     "create-post": "Create a help post on the forum",
     "cancel": "Cancel",
     "reset-lesson": "Reset this lesson",
+    "revert": "Revert",
+    "revert-to-saved-code": "Revert to saved code",
```
Comment (moT01) [inline client/i18n/locales/english/translations.json:56]:
```suggestion
    "revert-to-saved-code": "Revert to Saved Code",
```

I think this matches the other button text.

File: client/i18n/locales/english/translations.json [hunk 2]
```diff
@@ -416,6 +418,7 @@
     "reset": "Reset this lesson?",
     "reset-warn": "Are you sure you wish to reset this lesson? The editors and tests will be reset.",
     "reset-warn-2": "This cannot be undone.",
+    "revert-warn": "Are you sure you wish to revert this lesson? Your latest changes will be undone and the code reverted to the most saved version.",
```
Comment (moT01) [inline client/i18n/locales/english/translations.json:421]:
```suggestion
    "revert-warn": "Are you sure you wish to revert this lesson? Your latest changes will be undone and the code reverted to the most recently saved version.",
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Couple minor grammar suggestions. Other than that, LGTM 👍

**Review by moT01 — APPROVED** [review]

**Pull request merged**